src/hooks/useDownShift.ts

```diff
@@ -24,21 +24,35 @@ const useDownShift: UseDownShift = ({ setQuery, items, onSubmit, cioClient, prev
     itemToString: (item) => item?.value || '',
     onSelectedItemChange({ selectedItem }) {
       if (selectedItem) {
-        if (selectedItem?.section === 'Search Suggestions') {
-          setQuery(selectedItem.value || '');
-        }
         if (selectedItem?.value) {
           if (onSubmit) onSubmit({ item: selectedItem, originalQuery: previousQuery });
           try {
-            if (!selectedItem?.data?.url) {
+            if (selectedItem?.section === 'Search Suggestions') {
```

Comment:
I believe `!selectedItem?.data?.url` was to check if not product. But I believe it's more accurate to send search submit only if section is Search Suggestion